resources/test/tests/idlharness-primary-int-obj.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>idlharness: Primary interface</title>
+  <script src="../../testharness.js"></script>
```

Comment:
Would suggest absolute paths, here, for consistency with other tests and as per documentation (iirc).